src/ntid.ts

```diff
@@ -20,14 +20,28 @@ import invariant from 'invariant';
 import { v4 as uuidv4 } from 'uuid';
 
 export type ntid = string;
+export type ntidByteEncoding = Uint8Array;
```

Comment:
When I see a variable named "encoding" I think of something like "UTF-8". I think it's fine to omit this type unless we want to use opaque types.